src/schemas/generator/v1/loadZone.ts

```diff
@@ -27,23 +27,34 @@ export const AvailableLoadZonesSchema = z.enum([
 export const LoadZoneItemSchema = z.object({
   id: z.string(),
   loadZone: AvailableLoadZonesSchema,
-  percent: z.number().int().min(1, { message: 'Invalid percentage' }).max(100),
+  percent: z
+    .number({ message: 'Invalid percentage' })
+    .int()
+    .min(1, { message: 'Invalid percentage' })
+    .max(100, { message: 'Invalid percentage' }),
 })
 
 export const LoadZoneSchema = z.object({
   distribution: z.enum(['even', 'manual']),
-  loadZones: z.array(LoadZoneItemSchema).refine(
+  zones: z.array(LoadZoneItemSchema).refine(
     (data) => {
       if (data.length === 0) {
         return true
       }
 
-      const totalPercentage = data.reduce(
-        (sum, { percent }) => sum + percent,
-        0
-      )
+      const totalPercentage = currentLoadZonePercentage(data)
       return totalPercentage === 100
     },
-    { message: 'The sum of all distribution percentages must be 100' }
+    (data) => {
+      const totalPercentage = currentLoadZonePercentage(data)
+      return {
+        message: `Total percentage must be 100, currently ${totalPercentage}`,
+        path: ['root'],
```

Comment:
I took @Llandy3d's suggestion here from the previous PR and included the current percentage in the error message.

![Screenshot 2025-03-05 at 12 09 24 PM](https://github.com/user-attachments/assets/bcd6e844-83fa-470b-9d19-cc9c4a38d9dc)

Reply:
I wonder if it could be more readable as: 🤔 
`Total percentage must be 100 (currently 85)`